browser/ai_chat/page_content_fetcher_browsertest.cc

```diff
@@ -133,6 +133,22 @@ class PageContentFetcherBrowserTest : public InProcessBrowserTest {
     run_loop.Run();
   }
 
+  void ValidateOpenLeoButtonNonce(const base::Location& location,
+                                  const std::string& nonce,
+                                  bool expected_is_valid) {
```

Comment:
I think it would help readability if we had something like this if we dropped the `expected_is_valid` and just returned a bool, like:

```cxx
  bool ValidateOpenLeoButtonNonce(const base::Location& location,
                                  const std::string& nonce)
```

So rather than having:

```cxx
ValidateOpenLeoButtonNonce(FROM_HERE, "5566", false);
```

We would have:

```cxx
EXPECT_FALSE(ValidateOpenLeoButtonNonce(FROM_HERE, "5566"));
```

Reply:
https://github.com/brave/brave-core/pull/26082/commits/c82227cd44b8e34008aba9e0eb8d3fe607a2152c